File: java/trunk/TetramBDEJB/src/org/tetram/bdtheque/model/entity/DessinateurSerie.java
package org.tetram.bdtheque.model.entity;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

@Entity
@DiscriminatorValue("1")
public class DessinateurSerie extends AuteurSerie {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

}

File: bdtheque/php/trunk/src/BDTheque/DataModelBundle/Entity/DessinateurSerie.php
<?php

namespace BDTheque\DataModelBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * BDTheque\DataModelBundle\Entity\DessinateurSerie
 * 
 * @ORM\Entity
 */
class DessinateurSerie extends AuteurSerie {

    public function setSerie(Serie $serie) {
        $serie->addDessinateur($this);
        return parent::setSerie($serie);
    }

}

?>